crates/cheatcodes/src/inspector.rs

```diff
@@ -59,7 +59,7 @@ use revm::{
 use serde_json::Value;
 use std::{
     cmp::max,
-    collections::{BTreeMap, VecDeque},
+    collections::{BTreeMap, HashSet, VecDeque},
```

Comment:
import from alloy_primitives instead

Reply:
changed in https://github.com/foundry-rs/foundry/pull/10219/commits/7741ba5b25839b23e9f3cdf02a3344a099baa81f